quantum/rgb_matrix.c

```diff
@@ -68,6 +68,7 @@
 
 #ifndef EECONFIG_RGB_MATRIX
   #define EECONFIG_RGB_MATRIX EECONFIG_RGBLIGHT
+  #define EECONFIG_RGB_MATRIX_SPD EECONFIG_RGB_SPD
```

Comment:
Honestly, i think this should probably be outside of this ifdef, and in it's own.

But ... I'm not 100% sure about that